includes/contact_component.inc

```diff
@@ -178,6 +178,13 @@ function _webform_edit_civicrm_contact($component) {
     '#default_value' => $component['extra']['no_hide_blank'],
     '#parents' => array('extra', 'no_hide_blank'),
   );
+  $form['field_handling']['submit_disabled'] = array(
+    '#type' => 'checkbox',
+    '#title' => t("Submit disabled field value(s)"),
+    '#description' => t('Store disabled field value(s) in webform submissions.'),
+    '#default_value' => $component['extra']['submit_disabled'],
```

Comment:
This will produce an undefined index - you need to add the default for this field to the function `_webform_defaults_civicrm_contact()`